src/app/components/EmptyBlock/EmptyBlock.tsx

```diff
@@ -2,13 +2,16 @@ import React from "react";
 
 type EmptyBlockProps = {
 	className?: string;
-	message: string;
+	children?: React.ReactNode;
+	message?: string;
 };
 
-export const EmptyBlock = ({ className, message }: EmptyBlockProps) => (
+export const EmptyBlock = ({ className, children, message }: EmptyBlockProps) => (
 	<div
+		data-testid="EmptyBlock"
 		className={`border border-solid border-theme-neutral-300 dark:border-theme-neutral-800 p-6 rounded-lg text-theme-secondary-text ${className}`}
 	>
-		{message}
+		{children && <div>{children}</div>}
+		{message && <div>{message}</div>}
```

Comment:
There's no point in keeping `message` imho 🤔

Reply:
done @dated :+1: